x230/README.md

```diff
@@ -183,6 +183,9 @@ based supply for a second USB port (like [this](https://de.aliexpress.com/item/1
 	cd skulls-x230-<version>
 
 
+#### side note
+Flashing and reading with low speeds takes time. Be patient. E.g. unlocking the bottom chip with its two reads, one write and one verify step usually takes approximately one hour in total.
+
```

Comment:
isn't this only relevant when using a RPi for us? how about only adding "We flash at low speeds. Be patient. This may take one hour." in the RPi chapter (and not a new "side note" chapter)?

Thanks so far! I think we should add a note that this may take long, and if you really like this more, I can add this... Sorry if I'm a bit picky here:)